.ci.yaml

```diff
@@ -3978,13 +3978,32 @@ targets:
 
   - name: Mac_ios microbenchmarks_ios
     recipe: devicelab/devicelab_drone
-    presubmit: false
     timeout: 60
     properties:
       tags: >
         ["devicelab", "ios", "mac"]
       task_name: microbenchmarks_ios
 
+  - name: Mac_ios native_assets_ios_simulator
+    recipe: devicelab/devicelab_drone
+    presubmit: false
+    bringup: true # TODO(dacoharkes): Set to false in follow up PR and check that test works on CI.
+    timeout: 60
+    properties:
+      tags: >
+        ["devicelab", "ios", "mac"]
+      task_name: native_assets_ios_simulator
+
+  - name: Mac_ios native_assets_ios
+    recipe: devicelab/devicelab_drone
+    presubmit: false
+    bringup: true # TODO(dacoharkes): Set to false in follow up PR and check that test works on CI.
+    timeout: 60
+    properties:
+      tags: >
+        ["devicelab", "ios", "mac"]
+      task_name: native_assets_ios
+
```

Comment:
Oh wait, you do test it in the integration shard, right? My bad, I added this at the beginning of my review

Reply:
Although, looking at the README for integration.shard testing, I think you're only supposed to run against `flutter_tester` in the integration.shard

https://github.com/vashworth/flutter/blob/master/packages/flutter_tools/test/integration.shard/README.md

So maybe it would be better to make a macOS integration test

Reply:
@christopherfujino Do you have an opinion on this?

Reply:
> @christopherfujino Do you have an opinion on this?

It looks like this *is* using flutter-tester? https://github.com/flutter/flutter/pull/130494/files#diff-23a9696abb2839217a91641a47e161c2e7a3010989874a14f2ba0b78221d5d0fR28 Or were you referring to a different test?

Reply:
I can't tell what that link points to (maybe the PR is too big or something).

I was referring to this test, which tests both `flutter-tester` and the host device
https://github.com/flutter/flutter/blob/fae782f5a588cb08230ce082c8c67898b49defab/packages/flutter_tools/test/integration.shard/native_assets_test.dart#L27-L30

Reply:
testing a desktop app is not an issue, this LGTM (if I had to guess, it would be that the README in question is saying that you need the devicelab to run a test with a phone).